android-json-form-wizard/src/main/java/com/vijay/jsonwizard/utils/Utils.java

```diff
@@ -167,8 +173,41 @@ public static void hideProgressDialog() {
 
     public static int pixelToDp(int dpValue, Context context) {
         float dpRatio = context.getResources().getDisplayMetrics().density;
-        float pixelForDp =  dpValue * dpRatio;
+        float pixelForDp = dpValue * dpRatio;
 
         return (int) pixelForDp;
     }
+
+    /**
+     * Loads the strings property files used to keep the strings to be displayed on the forms
+     *
+     * @param key     {@link String}
+     * @param context {@link Context}
+     * @return property {@link String}
+     */
+    public static String getProperty(String key, Context context) {
+        Properties properties = new Properties();
```

Comment:
Consider using resource bundles which have been developed with Internalization in mind, check out https://developer.android.com/reference/java/util/ResourceBundle
https://developer.android.com/reference/java/util/PropertyResourceBundle.html